tests/runners/update_submission_runner.py

```diff
@@ -103,6 +105,9 @@ def run_primary_submission(self, dataset_name):
         # Disable indexing since this is an internal test for ingest, we don't need to trigger analysis pipelines
         submission_manager.submission_envelope.disable_indexing()
 
+        submission_manager.graph_validate_envelope()
+        submission_manager.wait_for_envelope_to_be_graph_valid()
```

Comment:
could rename the methods to
`validate_envelope_graph`or `validate_graph`
 `wait_for_envelope_to_have_valid_graph` for readability

Reply:
Done!